### DISCUSSION TITLE
Moving between hotspots?

### DISCUSSION THREAD

**Discussion by Tonikuu** [google/model-viewer#2357] [Q&A]
Hi, I was wondering is it possible to move between hotspots? I need to put inside annotations a button that allows movement between hotspots. 

Thanks for help!

**Comment by LaloYanez** [comment]
Indeed, you can give the annotations event listeners so they show extra information on click or move the camera to a custom location. You can do it by changing the camera-orbit, camera-target and field-of-view attributes as described in the documentation here: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-cameraOrbit
